pkg/build/builder/docker.go

```diff
@@ -313,10 +314,8 @@ func (d *DockerBuilder) dockerBuild(dir string, tag string, secrets []buildapi.S
 	}
 	opts.NetworkMode = network
 	if len(resolvConfHostPath) != 0 {
-		cmd := exec.Command("chcon", "system_u:object_r:svirt_sandbox_file_t:s0", "/etc/resolv.conf")
-		err := cmd.Run()
-		if err != nil {
-			return fmt.Errorf("unable to set permissions on /etc/resolv.conf: %v", err)
+		if err := label.Relabel(resolvConfHostPath, "system_u:object_r:svirt_sandbox_file_t:s0", true); err != nil {
```

Comment:
i don't think using the resolvConfHostPath is going to work here...you're running inside a container, the path needs to be relative to the container's filesystem, no?

Reply:
Ok so this path is not available in the container.  I will change back to /etc/resolv.conf